src/Gui/BitmapFactory.h

```diff
@@ -78,12 +78,14 @@ class GuiExport BitmapFactoryInst : public Base::Factory
     /** Retrieves a pixmap by name and size created by an
      * scalable vector graphics (SVG).
      */
-    QPixmap pixmapFromSvg(const char* name, const QSize& size) const;
+    QPixmap pixmapFromSvg(const char* name, const QSizeF& size,

```

Comment:
The std::map should be called by const reference. Instead of unsigned long using uint32_t should be sufficient. Also the doc string should be extended to explain what colorMapping is good for.

Reply:
Like this?

Reply:
Yes.

Reply:
Of course, the default arguments must be changed to std::map<uint32_t, uint32_t>(), too. So, right now I am building your branch locally and make the needed fixes to get it working...

Reply:
I realized that for the color mapping of the sketcher cursors unsigned long was already used before. So, to avoid to complicate things it's best to switch back to unsigned long here, too.
My actual point was that for the encoding of rgba colors we don't need an unsigned long which requires 8 bytes on UNIX-like systems (while on Windows it's 4 bytes only). So, sorry for the noise.